zebedee-reader/src/main/java/com/github/onsdigital/zebedee/content/partial/AlertType.java

```diff
@@ -1,6 +1,10 @@
 package com.github.onsdigital.zebedee.content.partial;
 
+import com.google.gson.annotations.SerializedName;
+
 public enum AlertType {
-    alert,
-    correction,
+    @SerializedName("alert")
+    ALERT,
+    @SerializedName("correction")
```

Comment:
Do you also want to add the get label functionality to this type as well?

Reply:
There is no need for that get label method here as this is only used for serialising/deserialising. I had to add the method in the page type because it is widely used somewhere else and needed to have a way of retrieving the value